cpp/autosar/src/rules/A5-2-6/OperandsOfALogicalAndOrNotParenthesized.ql

```diff
@@ -17,11 +17,17 @@
 import cpp
 import codingstandards.cpp.autosar
 
-from BinaryLogicalOperation op, BinaryOperation binop
```

Comment:
Given the query, I don't see how the false positives that were reported can occur. Let's exclude this query for now.